tests/utils/test_parse.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for the parse utility module."""
+
+import hail as hl
+import pytest
+
+from gnomad.utils.parse import parse_variant
+
+
+class TestParseVariant:
+    """Test the parse_variant function."""
+
+    grch37_out = hl.Struct(locus=hl.Locus("1", 1000, "GRCh37"), alleles=["A", "T"])
+    grch38_out = hl.Struct(locus=hl.Locus("chr1", 1000, "GRCh38"), alleles=["A", "T"])
+
+    @pytest.mark.parametrize(
+        "variant_str, contig, position, ref, alt, build, expected",
+        [
```

Comment:
I've never used pytest myself, you only need to put some representative valid and invalid sennarios, right?

Reply:
Yeah, it's just adding unit tests so that if we change the code and it breaks it, then hopefully the test will fail